lttng-record-trace

```diff
@@ -159,13 +160,21 @@ def do_trace(cmd, session, options):
                 cmd_stub([LTTNG, "enable-event", "-c", "k", "-a", "-k", "--syscall"], False)
             if profile["kernel"]:
                 cmd_stub([LTTNG, "enable-event", "-c", "k", "-k", ",".join(evlist)], True)
-    if (profile["ust"] or options.enable_all_events):
+    if (profile["ust"] or profile["jul"] or options.enable_all_events):
         cmd_stub([LTTNG, "enable-channel", "u", "-u", "--subbuf-size", "1024K", "--num-subbuf", "8"], True)
         if (options.enable_all_events):
             cmd_stub([LTTNG, "enable-event", "-c", "u", "-a", "-u"], True)
         else:
-            logging.info("userspace event list: " + ", ".join(profile["ust"]))
-            cmd_stub([LTTNG, "enable-event", "-c", "u", "-u", ",".join(profile["ust"])], True)
+            if profile["ust"]:
+                logging.info("userspace event list: " +
+                             ", ".join(profile["ust"]))
+                cmd_stub([LTTNG, "enable-event", "-c", "u", "-u",
+                          ",".join(profile["ust"])], True)
+            if profile["jul"]:
+                logging.info("JUL event list: " +
+                             ", ".join(profile["jul"]))
+                cmd_stub([LTTNG, "enable-event", "-c", "u", "--jul",
```

Comment:
For JUL, there is only one channel named 'lttng_jul_channel', so the channel parameter here has no effect. To modify the subbuf-size or num-subbuf, we need to configure a channel (with enable-channel), but name it with the default name lttng_jul_channel... I'll fix it in a later patch